src/InputWithLabel.js

```diff
@@ -0,0 +1,26 @@
+import React, { useEffect, useRef } from "react";
+
+function InputWithLabel ({id, value, inputChange, children}) {
+    const inputRef = useRef(null);
+
+    useEffect(() => {
+        inputRef.current.focus();
+    }, []);
+
+    return (
+        <>
+            <label htmlFor={id}>Title:{children}</label>
```

Comment:
In reference to my previous comment, we don't need the title here because it's actually what we're passing in as `children`